[16.0][FIX] payroll: Change mail template expressions

victoralmau opened the pull request:
FWP from 15.0: https://github.com/OCA/payroll/pull/124

Change mail template expressions according to 15.0

Please @pedrobaeza can you review it?

@Tecnativa

victoralmau committed:
[FIX] payroll: Change mail template expressions

pedrobaeza commented on payroll/__manifest__.py:
```diff
@@ -2,7 +2,7 @@
 {
     "name": "Payroll",
-    "version": "16.0.1.0.0",
+    "version": "16.0.2.0.0",
```
I think such high jump is not needed for this fix. I know that contributing guidelines expresses that it should be major when migrations scripts are needed, but it's referring to huge database model changes, not when we need to load an XML.

pedrobaeza commented:
Check CI

victoralmau committed:
[FIX] payroll: Fix tests since https://github.com/odoo/odoo/commit/09091dded0115c3af1e8730885fa2f018d17d128

Now a contract is closed whenever appropriate, therefore, the expected
value must be changed in tests.

nimarosa commented:
/ocabot merge nobump
